docs/API/imqsauth_api.yaml

```
@@ -954,6 +954,32 @@ paths:
               schema:
                 type: string
                 example: Error message
+  /msaad/approles:
+    get:
+      summary: Get MS AAD App Roles
+      security:
+        - basicAuth: []
+      responses:
+        '200':
+          description: OK
+          content:
+            text/plain:
+              schema:
+                type: string
```

Please and thank you
```suggestion
                type: array
                items:
                    type: string
```